doc/NodeCounters.md

```diff
@@ -0,0 +1,51 @@
+# NodeCounter
+## Per-counter
+### Cheap
```

Comment:
The "Cheap" and "Costly" sections in doc/NodeCounters.md are obsolete now ?

Reply:
Done